framework/src/modules/chain/loader.js

```diff
@@ -195,8 +194,14 @@ class Loader {
 			procedure: 'getSignatures',
 		});
 
-		const validate = promisify(this.schema.validate.bind(this.schema));
-		await validate(result, definitions.WSSignaturesResponse);
+		const validate = validator.validate(
+			definitions.WSSignaturesResponse,
+			result
+		);
+
+		if (!validate) {
```

Comment:
In some places we're using `if (!validate)` and in other places `if (validator.errors)`.
Prefer using it consistently.

Reply:
Tried to maintain the changes low due to four PRs changing the same code making merges very complex and prone to errors. Will wait for other PRs to merge before making this change.

Reply:
Found this case only here so I went ahead and replaced. Let me know if you spotted more cases I couldn't find any more. Resolving.